homeassistant/generated/integrations.json

```diff
@@ -1380,8 +1380,8 @@
       "integrations": {
         "emoncms": {
           "integration_type": "hub",
-          "config_flow": false,
-          "iot_class": "local_polling",
+          "config_flow": true,
+          "iot_class": "cloud_polling",
```

Comment:
Same query as in the manifest.json.
By default EmonCMS would be used with local hardware/server, so local_polling.
Similar to MQTT, which mostly is local, but can be used with a cloud service.